framework-core/src/main/java/org/daisy/pipeline/script/impl/parser/StaxXProcScriptParser.java

```diff
@@ -464,13 +464,13 @@ else if (elemsToIgnore == 0) {
 									else if (role.equals(Values.NAME)) {
 										reader.next();
 										dHolder.mShort = reader.peek()
-												.asCharacters().getData();
+												.asCharacters().getData().replaceAll("\\s\\s+","");
 									}
```

Comment:
With this regex, if I have an attribute `name="my    script"` then `mShort`'s value will be `myscript`, i.e. the space will be discarded entirely; shouldn't it be `my short` instead?
If the intent is really to replace sequences of more than one space by a single space, the code should be `.replaceAll("\\s+"," ")`, no?